.github/actions/go-tools-cache/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Load Go tools cache'
```

Comment:
Multi purpose cache, should have plenty cache hits even test cache was not triggered yet.